thread/common/thrprof.c

```diff
@@ -1025,3 +1030,67 @@ omrthread_get_jvm_cpu_usage_info_error_recovery(void)
 		GLOBAL_UNLOCK_SIMPLE(lib);
 	}
 }
+
+intptr_t
+omrthread_get_thread_times(omrthread_thread_time_t *threadTime)
+{
+#if defined(LINUX)
+	struct rusage rUsage;
+	memset(&rUsage, 0, sizeof(rUsage));
+
+	if (0 == getrusage(RUSAGE_THREAD, &rUsage)) {
+		threadTime->userTime = (SEC_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_utime.tv_sec)
+			+ (MICRO_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_utime.tv_usec);
+		threadTime->sysTime = (SEC_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_stime.tv_sec)
+			+ (MICRO_TO_NANO_CONVERSION_CONSTANT * (int64_t)rUsage.ru_stime.tv_usec);
+
+		return 0;
+	}
+
+	return -1;
+#elif defined(OMR_OS_WINDOWS) && !defined(BREW) /* defined(LINUX) */
+	omrthread_t self = omrthread_self();
+	FILETIME creationTime;
+	FILETIME exitTime;
+	FILETIME kernelTime;
+	FILETIME userTime;
+	memset(&creationTime, 0, sizeof(creationTime));
+	memset(&exitTime, 0, sizeof(exitTime));
+	memset(&systemTime, 0, sizeof(systemTime));
```

Comment:
```
D:\a\1\s\thread\common\thrprof.c(1059,20): error C2065: 'systemTime': undeclared identifier [D:\a\1\s\build\thread\j9thr_obj.vcxproj]
D:\a\1\s\thread\common\thrprof.c(1059,42): error C2065: 'systemTime': undeclared identifier [D:\a\1\s\build\thread\j9thr_obj.vcxproj]
```
```suggestion
	memset(&kernelTime, 0, sizeof(kernelTime));
```

Reply:
My bad, should've check it when pasting from elsewhere. Fixed now.